confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -301,17 +301,11 @@ def get_compatibility(self, subject=None):
         Get the current compatibility level for a subject.  Result will be one of:
 
         @:param: subject: subject name
-        @:return: 'NONE','FULL','FORWARD', or 'BACKWARD'
+        @:return: True if compatible, False if not compatible
```

Comment:
Let's follow what the Java code does:
https://github.com/confluentinc/schema-registry/blob/1443a7f8e251c93987099b26b0917bcebd4deca5/client/src/main/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClient.java#L315

 /config returns a JSON  result containing "compatibility". getCompatibilityLevel() wraps the result and returns that same "compatibility" (and not "compatibilityLevel"), unfortunate naming perhaps.

And I propose raising an exception if code != 2xx or if the returned value is not one of the four strings.